docs/_articles/en/v3.x/release-information.md

```diff
@@ -12,6 +12,32 @@ We publish the latest Code Analyzer monthly.
 * Run `sfdx plugins:update` and `sfdx plugins:update --help` to update Code Analyzer and help to the latest version.
 * Follow these [instructions](./en/v3.x/getting-started/install/#upgrade-plug-in) to update Code Analyzer
 
+## [v3.6.0](https://github.com/forcedotcom/sfdx-scanner/tree/v3.6.0) (10-25-2022)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v3.5.1...v3.6.0)
+
+### Release Summary
+
+* NEW: SFCA v3.x will now publish as `latest`.
+* CHANGE \[SFGE\]: `scanner:run:dfa` has a new `--rule-disable-warning-violation` flag to disable warnings on StripInaccessible call with AccessType.READABLE.
```

Comment:
Since run:dfa has a new flag ... shouldn't this be NEW instead of CHANGE?

EDIT -->

NEW: Use the `--rule-disable-warning-violation` flag with `scanner:run:dfa` to disable warnings on StripInaccessible calls with AccessType.READABLE.